#11 [ADD] SPN 기능 구현

KWY0218 opened the pull request:
history 에 함수 2개 추가 했음

프로세스 스케줄링이 끝난 마지막에  전력량을 체크하는 방식이던데
나는 1초마다 코어의 전력량을 증가 시키는 쪽으로 구현했음

KWY0218 committed:
#11 [ADD] SPN 기능 구현

history 에 함수 2개 추가 했음
프로세스 스케줄링이 끝난 마지막에  전력량을 체크하는 방식이던데
나는 1초마다 코어의 전력량을 증가 시키는 쪽으로 구현했음

Dev-SaTu reviewed (changes requested):
History가 아니라 Core에 함수 두 개 추가된 것 같아요
1. 코어 클래스에 변수 추가하지 말고
2. 대신 History 클래스에 누적 전력소모량 List<Double> 두는 건 어떤가요

Dev-SaTu committed:
#18 [REFACTOR] Process

KWY0218 committed:
[REFACTOR] SPN 수정

accConumption 부분 수정
SPN 스케줄 다음 History 에 totalAccConsumption, totalBurstTime 설정
